requirements.txt

```diff
@@ -0,0 +1,6 @@
+importlib-metadata==4.13.0
+kf_lib_data_ingest @ git+https://github.com/kids-first/kf-lib-data-ingest.git@latest-release
+kf_utils @ git+https://github.com/kids-first/kf-utils-python.git@latest-release
```

Comment:
We can potentially do this in another PR, but I'm thinking it might be good to setup some CI where we can do a basic integration test of the ETL so that we can more quickly catch these types of issues. The CI would do the following:

- Pull in the the docker image and setup the KF Dataservice container
- Pull in the the docker image and setup the KF FHIR service container
- Populate the dataservice with some sample data for the ETL
- Setup python environment
- Run ETL
- Check that the FHIR service has the expected loaded entities

Reply:
I updated the `requirements.txt` file as suggested and created a GitHub issue (#8) to keep track of CI addition.